coredhcp.go

```diff
@@ -104,20 +104,26 @@ func (s *Server) LoadPlugins(conf *config.Config) ([]*plugins.Plugin, []*plugins
 // It will not reply if the resulting response is `nil`.
 func (s *Server) MainHandler6(conn net.PacketConn, peer net.Addr, req dhcpv6.DHCPv6) {
 	var (
-		resp, tmp dhcpv6.DHCPv6
-		stop      bool
-		err       error
+		resp dhcpv6.DHCPv6
+		stop bool
+		err  error
 	)
 
 	// Create a suitable basic response packet
-	switch req.Type() {
+	tmp := req
```

Comment:
Is this `tmp` variable actually needed? I think that with the `req` and `resp` it should be enough.

Reply:
good point, it's not really needed. I was keeping it for the rainy days in case we wanted the original packet :) but until it's needed, it's not needed

Reply:
actually, I realized why I did that: it's due to an unrelated change that I will make in another PR (since I'm backporting the changes that I made for the LinuxBoot demo), I'll probably have to reintroduce it later, but for it's good as you suggest